lib/rucio/core/transfer.py

```diff
@@ -45,9 +46,10 @@
 REGION_SHORT = make_region().configure('dogpile.cache.memcached',
                                        expiration_time=600,
                                        arguments={'url': "127.0.0.1:11211", 'distributed_lock': True})
+USER_TRANSFERS = config_get('conveyor', 'user_transfers', False, None)
```

Comment:
@vingar Just to keep track and summarize the discussion on slack about this point. The idea here is to optimize the mapping at a second round of iteration (dedicated issue for that? what do you think?) since it involves the splitting of transfers by scope that could be potentially dangerous to include for the production conveyor without proper tests. Is this correct?

Reply:
👍 Fully agree to the iterative implementation.